test/benchmark/bench_linux_and_aster.sh

```diff
@@ -10,34 +10,52 @@ command -v jq >/dev/null 2>&1 || { echo >&2 "jq is not installed. Aborting."; ex
 
 # Script directory
 BENCHMARK_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
-# Kernel image 
-KERNEL_DIR="/root/dependency"
-LINUX_KERNEL="${KERNEL_DIR}/vmlinuz"
-KERNEL_VERSION="5.15.0-105-generic"
-MODULES_DIR="${BENCHMARK_DIR}/../build/initramfs/lib/modules/${KERNEL_VERSION}/kernel"
+# Dependencies for Linux
+LINUX_DEPENDENCIES_DIR="/opt/linux_dependencies"
+LINUX_KERNEL="${LINUX_DEPENDENCIES_DIR}/vmlinuz"
+LINUX_KERNEL_VERSION="5.15.0-105-generic"
+LINUX_MODULES_DIR="${BENCHMARK_DIR}/../build/initramfs/lib/modules/${LINUX_KERNEL_VERSION}/kernel"
 # Atomic wget script
 WGET_SCRIPT="${BENCHMARK_DIR}/../../tools/atomic_wget.sh"
 
-# Generate entrypoint script for Linux cases
-# TODO: Disable optimize-related features in Linux's ext2 using 'mkfs.ext2 -O ^[feature]'
-generate_entrypoint_script() {
+# Prepare Linux kernel and modules
+prepare_libs() {
+    if [ ! -f "${LINUX_KERNEL}" ]; then
+        echo "Downloading the Linux kernel image..."
+        mkdir -p "${LINUX_DEPENDENCIES_DIR}"
+        ${WGET_SCRIPT} "${LINUX_KERNEL}" "https://raw.githubusercontent.com/asterinas/linux_kernel/9e66d28/vmlinuz-${LINUX_KERNEL_VERSION}" 
+    fi
+    if [ ! -d "${LINUX_MODULES_DIR}" ]; then
+        echo "Downloading additional kernel modules..."
+        mkdir -p "${LINUX_MODULES_DIR}/drivers/block"
```

Comment:
Kernel modules are not cached. If the kernel itself should be cached, why not the kernel modules?

Reply:
This is a left-over introduced by others. I think I should handle it in this PR.